AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -301,7 +301,6 @@ - (void)testLogsNotFlushedImmediatelyWhenIntervalIsCustom {
   [self waitForExpectationsWithTimeout:kMSTestTimeout
                                handler:^(NSError *error) {
                                  OCMVerify([self.sut startTimer:OCMOCK_ANY]);
-                                 assertThatUnsignedLong(self.sut.itemsCount, equalToInt(itemsToAdd));
```

Comment:
Please do not include not related changes to this PR. Add TODO or FIXME comment at least.